ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,59 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+
+def _multiple_shape_helper(draw):
+
+    dtypes_and_input = draw(
+        helpers.dtype_and_values(
+            available_dtypes = helpers.get_dtypes("numeric")))
+    
+    #input = dtypes_and_input[1][0]
+    input_dims = len(ivy.shape(dtypes_and_input[1][0]))
+    
+    
+    shape = draw(helpers.get_shape(
+            min_num_dims=1, max_num_dims=1, min_dim_size=input_dims, max_dim_size=input_dims
+        ))
+    dt_and_multiples = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            shape=shape))
+    
+    return dtypes_and_input, dt_and_multiples
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input, dt_and_multiples=_multiple_shape_helper(),
+    ),
```

Comment:
The `),` is a syntax error.

Reply:
thank you, this one had me questioning my sanity

Reply:
I think so.

Reply:
We don't unpack inside the handle test decorators. Just make one argument `dtypes_input_multiples` and unpack it inside the tester function.